Toolkit/ArcGISToolkit/MeasureToolbar.swift

```diff
@@ -266,7 +278,9 @@ public class MeasureToolbar: UIView, AGSGeoViewTouchDelegate {
         segControl.setImage(measureFeatureImage, forSegmentAt: 2)
         segControlItem = UIBarButtonItem(customView: segControl)
         
-        resultView.buttonTapHandler = unitsButtonTap
```

Comment:
This was giving the result view a strong reference to the measure toolbar, thereby creating a retain cycle and preventing the toolbar from being deallocated.